include/swift/Option/Options.td

```diff
@@ -1033,6 +1033,10 @@ def disable_parser_lookup : Flag<["-"], "disable-parser-lookup">,
   Flags<[FrontendOption]>,
   HelpText<"Disable parser lookup & use ast scope lookup only (experimental)">;
 
+def disable_request_based_incremental_dependencies : Flag<["-"], "disable-request-based-incremental-dependencies">,
+  Flags<[FrontendOption]>,
+  HelpText<"Disable request-based name tracking">;
+
```

Comment:
Even though it seems redundant, it is also good to have an "enable" option, as we do for others. That way, if we have to change the default to disable, we can still test with enabled.